src/SearchResults.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import TableHeader from "./TableHeader";
+import TableData from "./TableData";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
```suggestion
import fakeBookings from "./data/fakeBookings.json";
```

Reply:
Stick to lowercase here for the first letter - I'd save variable names starting with a capital for React components only.